### ISSUE TITLE
Image alt text

### ISSUE THREAD

**Issue by freemvmt** [bethanyios/personalWebsite#3]
> Image elements do not have [alt] attributes

Images should have alt texts to be read by screen readers etc :)
e.g. profile and slider photos
